apps/block_scout_web/assets/js/pages/address/forward_transfers.js

```diff
@@ -0,0 +1,119 @@
+import $ from 'jquery'
+import omit from 'lodash.omit'
+import humps from 'humps'
+import numeral from 'numeral'
+import socket from '../../socket'
+import { batchChannel } from '../../lib/utils'
+import { connectElements } from '../../lib/redux_helpers.js'
+import { createAsyncLoadStore } from '../../lib/async_listing_load'
+import '../address'
+import { isFiltered } from './utils'
+
+const BATCH_THRESHOLD = 10
+
+export const initialState = {
+  channelDisconnected: false,
+  addressHash: null,
+  filter: null,
+  forwardTransfersBatch: []
+}
+
+export function reducer (state, action) {
+  switch (action.type) {
+    case 'PAGE_LOAD':
+    case 'ELEMENTS_LOAD': {
+      return Object.assign({}, state, omit(action, 'type'))
+    }
+    case 'CHANNEL_DISCONNECTED': {
+      if (state.beyondPageOne) return state
+
+      return Object.assign({}, state, {
+        channelDisconnected: true,
+        forwardTransfersBatch: []
+      })
+    }
+    case 'RECEIVED_NEW_FORWARD_TRANSFER_BATCH': {
+      if (state.channelDisconnected || state.beyondPageOne) return state
+
+      const incomingforwardTransfers = action.msgs
+        .filter(({ toAddressHash, fromAddressHash }) => (
+          !state.filter ||
```

Comment:
same question from `fee_payments.js`: is intended in that we want to return `action.msgs` elements that don't have the `state.filter` value set?